src/main/resources/plugin.yml

```diff
@@ -1,9 +1,10 @@
 name: TradeShop
-version: 1.8.1
+version: 2.0.0-DEV
```

Comment:
same here, ought to remove the `DEV` tag before releasing

Reply:
added todos in TradeShop so that we wont forget, hopefully